src/components/ProgressBar/ProgressBar.scss

```diff
@@ -0,0 +1,32 @@
+.progress-bar {
+  display: flex;
+  flex-direction: column;
+
+  label.bar-label {
+    font-size: 10px;
+    white-space: nowrap;
+  }
+
+  .outer-bar {
+    background-color: #ffff;
+    border: 1px solid #25283d;
+    border-radius: 10px;
+    height: 10px;
+    width: 100%;
+
+    .inner-bar {
```

Comment:
There's a bit of a UI bug here since the `outer-bar` has a white background, it leaves this white-space:
![image](https://user-images.githubusercontent.com/29434693/159387837-4941a7f6-f257-4526-b6ed-7f54397df5f0.png)

A proposed fix is to inherit the border and set a negative margin in `inner-bar` of the same size to offset the border swallowing the space:
```css
.inner-bar {
    border: inherit;
    margin: -1px;    // should be same size as border
    min-width: 10px; // increase a little to offset smaller margin
}
```

Results in:
![image](https://user-images.githubusercontent.com/29434693/159388001-85282045-6124-4c8d-bdb6-2727f6c6a816.png)

Reply:
To add to this, I noticed that setting the progress bar to `0%` still has this filled due to the `min-width`.
We could potentially hard code `min-width: 0` in the code if `percent = 0`, but up to your judgment.

Reply:
Thanks for catching that, haha. I got it changed with your suggestions. 

When I set the min-width to 0, the inner bar still appeared for me and broke through the outer bar. 

I added some logic for ProgressBar to not render the inner bar when the percent is 0. 
```
{percent > 0 && <div className="inner-bar"> ... }
```